server/entity/User.ts

```diff
@@ -25,14 +26,19 @@ export class User {
 
   static readonly filteredFields: string[] = ['plexToken', 'password'];
 
+  public displayName: string;
+
   @PrimaryGeneratedColumn()
   public id: number;
 
   @Column({ unique: true })
   public email: string;
 
-  @Column()
-  public username: string;
+  @Column({ nullable: true })
+  public plexUsername: string;
+
+  @Column({ type: 'text', nullable: true })
```

Comment:
Why the type of `text` here instead of `varchar`?

Reply:
I don't recall why I wanted to try this. Removed it now.